Restructure AI skills docs: replace Approach 1/2 with Skill Locations reference and Installing Skills (#1944)
IgniteUI/igniteui-xplat-docs: doc/en/components/ai/skills.md

<!-- WebComponents -->

```
.agents/
  skills/
    igniteui-wc-choose-components/
      SKILL.md
    igniteui-wc-integrate-with-framework/
      SKILL.md
    igniteui-wc-customize-component-theme/
      SKILL.md
    igniteui-wc-optimize-bundle-size/
      SKILL.md
```

<!-- end: WebComponents -->

<!-- React -->

```
.agents/
  skills/
    igniteui-react-components/
      SKILL.md
    igniteui-react-customize-theme/
      SKILL.md
    igniteui-react-optimize-bundle-size/
      SKILL.md
```

<!-- end: React -->

For user-level (global) skills available across all projects, use `~/.agents/skills/` instead.

### GitHub Copilot

[GitHub Copilot](https://docs.github.com/en/copilot/concepts/agents/about-agent-skills) discovers skills from:

| Scope | Location |
|:------|:---------|
| Project | `.github/skills/`, `.claude/skills/` |
| Personal | `~/.copilot/skills/`, `~/.claude/skills/` (Copilot coding agent and GitHub Copilot CLI only) |

> > **Tip:** In VS Code, you can configure additional skill locations (including `.agents/skills/`) using the `chat.agentSkillsLocations` setting.

Reviewer: ```suggestion
> **Tip:** In VS Code, these locations also include the general  `.agents/skills/`/`~/.agents/skills/` and you can configure additional skill locations using the `chat.agentSkillsLocations` setting.
```

<!-- WebComponents -->

```
.agents/
  skills/
    igniteui-wc-choose-components/
      SKILL.md
    igniteui-wc-integrate-with-framework/
      SKILL.md
    igniteui-wc-customize-component-theme/
      SKILL.md
    igniteui-wc-optimize-bundle-size/
      SKILL.md
```

<!-- end: WebComponents -->

<!-- React -->

```
.agents/
  skills/
    igniteui-react-components/
      SKILL.md
    igniteui-react-customize-theme/
      SKILL.md
    igniteui-react-optimize-bundle-size/
      SKILL.md
```

<!-- end: React -->

For user-level (global) skills available across all projects, use `~/.agents/skills/` instead.

### GitHub Copilot

[GitHub Copilot](https://docs.github.com/en/copilot/concepts/agents/about-agent-skills) discovers skills from:

| Scope | Location |
|:------|:---------|
| Project | `.github/skills/`, `.claude/skills/` |
| Personal | `~/.copilot/skills/`, `~/.claude/skills/` (Copilot coding agent and GitHub Copilot CLI only) |

> > **Tip:** In VS Code, these locations also include the general  `.agents/skills/`/`~/.agents/skills/` and you can configure additional skill locations using the `chat.agentSkillsLocations` setting.

Reviewer: ```suggestion
> **Tip:** In [VS Code](https://code.visualstudio.com/docs/copilot/customization/agent-skills), these locations also include the general  `.agents/skills/` and `~/.agents/skills/` and you can configure additional skill locations using the `chat.agentSkillsLocations` setting.
```

The CLI will guide you through a series of prompts to:

1. Select which skills to install (components, theming, etc.).

> 2. Choose the target location for the skill files in your project (e.g. `.agents/skills/`, `.github/skills/` — see [Skill Locations](#skill-locations) for all options).

Reviewer: ```suggestion
2. Choose the target location for the skill files in your project (e.g. `.agents/skills/`, `.github/skills/`).
```
